be/src/common/config.h

```diff
@@ -1130,4 +1130,6 @@ CONF_mInt64(lake_vacuum_min_batch_delete_size, "1000");
 // TOPN RuntimeFilter parameters
 CONF_mInt32(desc_hint_split_range, "10");
 
+CONF_mBool(use_lake_pk_index_loader, "true");
```

Comment:
this PR should be considered as behavior change or not?

Reply:
Changed to false